data-products.md

```diff
@@ -0,0 +1,37 @@
+# Data Products
+
+The following tables list available TERRA-REF datasets. The tables will be updated as new datasets are released.  Links are provided to pages with detailed information about each dataset.
+
+## Level 0
+
+Raw sensor data products used as inputs into computational processes to produce calibrated  \(Level 1\) and higher-level derived data products.  For more information about derived data products, see sections below.
+
+| Dataset | Category | Sensor |
+| --- | --- | --- |
+| [Fluorescence intesity imaging](/fluorescence-intensity-imaging.md) | Imaging | PSII Camera |
+| [Infrared data for heat imaging](/flir.md) | Imaging | FLIR |
+| Photochemical reflectance index \(PRI\) | Imaging | Skye PRI |
+| Short-wave infrared | Imaging | Headwall SWIR |
+| Visible light imaging \(stereo\) | Imaging | Prosilica |
+| Visible-near-infrared | Imaging | Headwall VNIR |
+| Atmospheric carbon dioxide | Environmental | Vaisala CARBOCAP |
+| Downwelling radiance | Environmental | Ocean Optics STS-VIS |
+| Downwelling photosynthetically active radiation | Environmental | Quantum PAR |
+| Environmental \(relative humidity, temperature, air pressure, wind, radiation, precipication\) | Environmental | Thies Clima |
+| Spectral reflectance | Environmental | Crop Circle |
+| Weather station | Environmental |  |
+
+## Level 1
+
+| Dataset | Description |
+| --- | --- |
+| Demosaic |  |
+| EnvironmentalLogger |  |
```

Comment:
EnvironmentalLogger is the name of a sensor suite ... the 'dataset' can be called 'weather' and / or 'environmental data'

Similarly, scanner3DTop could be something like '3D point cloud'

Demosaic ...?